rmgpy/data/kinetics/library.py

```diff
@@ -143,7 +143,6 @@ def getLibraryReactions(self):
             if entry._longDesc and 'Originally from reaction library: ' in entry._longDesc:
                 lib = entry._longDesc.replace('Originally from reaction library: ','')
                 lib = lib.replace('\n','')
-                logging.info(lib)
```

Comment:
Should these be logging.debug statements instead? (Lines 146, 155, 162)

Reply:
For stuff that occurs so early in simulation, (makes it easy to test) I'm comfortable requiring them to add their own debugging statements.  These statements were not particularly helpful when I was debugging the parsing issue and after thinking about it I've decided that they clutter up the log file too much to be worth keeping.